about/index.md

```diff
@@ -1,39 +1,72 @@
 
-{% include section.html %}
+Some of the specific things we work on are:
 
-## The Team
+- Modern, responsive web applications for data scientists, biologists, and more
+- Powerful and flexible backend systems and APIs for programmers
+- Interactive data visualizations for insights and engagement
+- Efficient and reproducible pipelines for analyzing large data
+- Scalable and robust cloud infrastructure
+- Beautiful websites with cohesive brand identities
+- Automations for processing, deployment, testing, monitoring, etc. that can save time, money, and hassle
 
-{% include list.html data="members" component="portrait" %}
+But the best way to understand the things we do is by looking at the code and using the software yourself:
+
+{%
+  include button.html
+  link="portfolio"
+  text="Our portfolio"
+  icon="fas fa-arrow-right"
+  flip=true
+  style="button"
+%}
 
 {% include section.html %}
 
-{% capture col1 %}
-{% include figure.html image="images/ahsb.jpg" %}
-{% endcapture %}
-{% capture col2 %}
-{% include figure.html image="images/anschutz.jpg" %}
-{% endcapture %}
+## Teaching and communication
+
+Whenever we can, we like to communicate our knowledge and skills to others.
+We believe this benefits the community we operate in and allows us to create better software together.
+
+On this website, we have a [blog](../blog) where we occasionally post tips, tricks, and other insights related to Git, workflows, code quality, and more.
+
+We have given workshops and personalized lessons related to Docker, cloud services, and more.
+We're always happy to set up a session to discuss technical trade whenever someone has the need.
+
+Here are some notable links to check out if you want to learn more from us:
+
+- ???
 
-{% include cols.html col1=col1 col2=col2 %}
+## Scope of our work
+
+Being central to the department, and not strictly associated with any particular lab or group within it, we need to ensure that we divide up our time and effort fairly.
+While we can do things like build full-stack apps from scratch and maintain complex infrastructure, the projects we take on tend to be small to medium size so that we leave ourselves available to others who need our help.
+Certain projects that are very large and long term in scope, such as ones that need to be HIPAA compliant, will fall outside of our purview and might lead you to hire a dedicated developer to fill your needs.
+That said, we can still provide partial support as a consulting body, a repository of information, a hiring advisor, and more.
+
+## Contact
+
+We are working on a more formal way to submit requests for support, but in the meantime, please contact [Audrey Wen](mailto:audrey.wen@cuanschutz.edu) as an entrypoint for requesting work from the Software Engineering Team.
+
+For general questions or technical help, we also have weekly open-office hours, Thursdays at 2:00 PM Mountain Time in [this Zoom room](https://ucdenver.zoom.us/j/94187504100).
```

Comment:
Wanted to double check: is this Zoom link password protected? If not, we should consider using one and spelling it out with natural language here to obfuscate automated "Zoom bombing" attempts. We could also use the "waiting room" capabilities to approve attendees individually as they show up (possibly even outside of the "office hours" duration?).